attributes/default.rb

```diff
@@ -67,7 +51,7 @@
 # This specifies how to connect to a master or a quorum of masters. This flag
 # works with 3 different techniques.  See Mesos documentation for details:
 # http://mesos.apache.org/documentation/latest/configuration
-default['mesos']['slave']['flags']['master']                = '`cat /etc/mesos/zk`'
+default['mesos']['slave']['flags']['master']                = 'localhost:5050'
```

Comment:
Same goes for this setting.  While the slave can find a master at a particular address and port you pretty much always want Mesos slaves to find their masters via zookeeper because that is what provides much of the fault tolerance that Mesos provides.  This will also probably make the serverspec tests fail.